### PR TITLE
Carousel spec

### PR DESCRIPTION
**Pull request by JeanRoca**
Uploaded carousel spec. This closes issue #6438 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6698)

### CODE REVIEW

File: source/nodejs/adaptivecards-site/pages/_posts/2021/Announcing-1.5.md [hunk 1]
```diff
@@ -62,4 +62,4 @@ These features are available today for you to test in our designer! We will keep
 
 ## Learn more
 
-Check out the [full release notes](https://github.com/microsoft/AdaptiveCards/releases/tag/21.10) to learn more and get started!
+Check out the [full release notes](https://github.com/microsoft/AdaptiveCards/releases/tag/21.09) to learn more and get started!
```
Comment (paulcam206) [inline source/nodejs/adaptivecards-site/pages/_posts/2021/Announcing-1.5.md:65]: probably don't want this change :)

**Review by paulcam206 — APPROVED** [review]
spec looks good, but we should probably not be updating the blog post :)

File: source/nodejs/adaptivecards-site/pages/_posts/2021/Announcing-1.5.md (hunk 1 above)
Comment (JeanRoca) [reply source/nodejs/adaptivecards-site/pages/_posts/2021/Announcing-1.5.md]: After an update to the fork this got fixed

**Comment by jwoo-msft** [comment]
this should be consolidated with this -> https://github.com/microsoft/AdaptiveCards/blob/feature/carousel/specs/DesignDiscussions/Carousel.md

**Review by jwoo-msft — CHANGES_REQUESTED** [review]
can you change the merging branch from `main` to `feature\carousel`?

**Review by jwoo-msft — APPROVED** [review]

**Pull request merged**